File: documentation/user/en/get-started/example/update-entity.cs
evita.UpdateCatalog(
	"testCatalog",
	session => {
		session.GetEntity(
			"Product", 1,
			AttributeContentAll(),
			PriceContentRespectingFilter()
		)
			.OpenForWrite()
			.SetAttribute("stockQuantity", 12)
			.SetPrice(
				1, "basic", new Currency("EUR"),
				51.64m, 22m, 63m,
				true
			)
			.UpsertVia(session);
	}
);

File: documentation/user/en/get-started/example/update-entity.java
evita.updateCatalog(
	"evita",
	session -> {
		session.getEntity(
			"Product", 1,
			attributeContentAll(),
			priceContentAll()
		)
			.orElseThrow()
			.openForWrite()
			.setAttribute("stockQuantity", 12)
			.setPrice(
				1, "basic", Currency.getInstance("EUR"),
				new BigDecimal("51.64"), new BigDecimal("22"), new BigDecimal(63),
				true
			)
			.upsertVia(session);
	}
);
